tasks/release.py

```diff
@@ -786,7 +786,7 @@ def update_build_links(_ctx, new_version, patch_version=False):
     content = confluence.get_page_by_id(page_id=BUILD_LINKS_PAGE_ID, expand="body.storage")
 
     title = content["title"]
-    current_version = title[-11:]
+    current_version = title.split()[-1].strip()
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
What is an example of `title`?

Reply:
The page title is supposed to be `Build links - <version>`